clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -269,7 +269,7 @@ public void onResponse(
         private void onFailure(final long currentTimeMs,
                                final Errors responseError) {
             log.debug("Offset fetch failed: {}", responseError.message());
-
+            requestState.onFailedAttempt(currentTimeMs);
```

Comment:
This is not needed because the only place for retry is when there's a COORDINATOR_LOAD_IN_PROGRESS and the onFailedAttempt is invoked in the retry.